webclient/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@aics/agave-webclient",
-  "version": "1.7.2",
+  "version": "1.8.0-rc.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "@aics/agave-webclient",
-      "version": "1.7.2",
+      "version": "1.8.0-rc.0",
```

Comment:
@toloudis this isn't supposed to auto-bump agave, right?

Reply:
I'm not sure how they did it, but I think this is actually ok in this case, because my automated version bump updates package.json but not package-lock.  I need to come up with a solution for this.